mercury_app/models.py

```diff
@@ -77,6 +78,23 @@ class Order(models.Model):
         default='PE',
     )
 
+    @property
+    def estate(self):
+        total_mercha = Merchandise.objects.filter(
+            order=self).values(name_count=Sum('quantity'))
+        ids_mercha = Merchandise.objects.filter(
+            order=self).values('id').order_by('id')
+        handed_mercha = Transaction.objects.filter(
```

Comment:
Can merge ids_mercha and handed_mercha in 1 query?

Reply:
Yes!